git-config/src/file/git_config.rs

```diff
@@ -1673,6 +1704,125 @@ mod from_paths {
         assert_eq!(config.len(), 1);
     }
 
+    #[test]
+    fn multiple_include() {
+        let dir = tempdir().unwrap();
+
+        let a_path = dir.path().join("a");
+        fs::write(
+            a_path.as_path(),
+            r"
+            [core]
+              a = false
+            ",
+        )
+        .expect("Unable to write config file");
```

Comment:
In tests, just unwrapping is fine in 'step one: getting it to work'. In 'step two: making it pretty' one can replace them with `?`.